easybuild/easyconfigs/r/ROOT/ROOT-6.14.06-foss-2018b-Python-3.6.6.eb

```diff
@@ -44,10 +44,9 @@ dependencies = [
 configopts = " -Dxrootd=OFF -Dmysql=OFF -Dkrb5=OFF -Dodbc=OFF -Doracle=OFF -Dpgsql=OFF -Dqt=OFF"
 
 # make sure some components are enabled
-configopts += " -Dpython=ON -Dpython3=ON -Dpcre=ON -Dzlib=ON"
 configopts += " -Dunuran=ON -Dtable=ON -Dexplicitlink=ON -Dminuit2=ON -Droofit=ON "
 
 # Add component-specific settings based on dependencies
-configopts += ' -Dfftw3=ON -Dgsl=ON -DOpenGL_GL_PREFERENCE=GLVND'
+configopts += ' -Dfftw3=ON -Dgsl_shared=ON -DOpenGL_GL_PREFERENCE=GLVND'
```

Comment:
@rarensu It looks like this part is missing in `ROOT-6.14.06-foss-2018b-Python-2.7.15.eb`, do you mind adding it there too while you're at it?

Reply:
Did I do that correctly?

Reply:
Yes, just adding another commit like you did is OK, but I would also make sure that the comments are the same in both 6.14.06 easyconfigs?